api/pom.xml

```diff
@@ -337,6 +338,7 @@
                         </goals>
                         <configuration>
                             <release>8</release>
+                            <createMissingPackageInfoClass>true</createMissingPackageInfoClass>
```

Comment:
we don't want to create "empty" files

Reply:
If I set it to false, it does not generate any package-info.class at all.

It should be fine to keep it as it is, as long as we don't have packages with no package-info.java

However this seems to be a bug with the maven compiler plugin, and this is a workaround to bypass that.

Reply:
It is set to false now, because as you told me, it does not create the .class but it generates the javadoc. And in case something is added in the package-info.java, it generates the package-info.class